### ISSUE TITLE
showing more subscription information in `--info`

### ISSUE THREAD

**Issue by arax** [hepix-virtualisation/vmcatcher#10]
The output of
```bash
vmcatcher_subscribe ... --info
```
should contain more information. In particular, information about the handling of new images (auto-accept, auto-subscribe, etc.) configurable with
```bash
  --imagelist-newimage-accept
                        Accept new image UUID in subscription.
  --imagelist-newimage-subscribe
                        Accept and subscribe to new image UUID in
                        subscription.
  --imagelist-newimage-refuse
                        Refuse new image UUID in subscription. (default)

```

**Comment by osynge** [comment]
This is a very good suggestion.

**Comment by osynge** [comment]
75a3650b1874f94990cc99261b14709758a26c16 is a proposed fix.

This will under go testing tonight.

**Comment by osynge** [comment]
resolved with 0.6.1 release
